src/data/Storage.ts

```diff
@@ -54,133 +54,147 @@ export const storage = {
 
 // NOTE: Predefined templates for test purpose
 //       for use just uncomment one of the following lines:
-// _emptyArrayOfCycles().catch((err) => console.error(err));
-// _arrayOfCyclesWithSingleCycle().catch((err) => console.error(err));
-// _arrayOfCyclesWithSingleCycleWithDelay().catch((err) => console.error(err));
-// _halfFilledArrayOfCycles().catch((err) => console.error(err));
-// _fullyFilledArrayOfCycles().catch((err) => console.error(err));
-// _fullyFilledArrayOfCyclesWithDelay().catch((err) => console.error(err));
+
+// _todayPeriod(6).catch((err) => console.error(err));
+// _todayOvulation(6).catch((err) => console.error(err));
+// _tomorrowOvulation(6).catch((err) => console.error(err));
+// _menstrualPhase(6).catch((err) => console.error(err));
+// _follicularPhase(6).catch((err) => console.error(err));
+// _lutealPhase(6).catch((err) => console.error(err));
+// _delayOfCycle(6).catch((err) => console.error(err));
 
 function _emptyArrayOfCycles(): Promise<void> {
   return storageImpl.remove("cycles") as Promise<void>;
 }
 
-function _arrayOfCyclesWithSingleCycle(): Promise<void> {
-  return storage.set.cycles([
-    {
+function _todayPeriod(countOfCycles: number): Promise<void> {
+  const date: Date = new Date();
+  date.setHours(0, 0, 0, 0);
+
+  const cycles: Cycle[] = [];
+
+  for (let i = 0; i < countOfCycles; ++i) {
+    date.setDate(date.getDate() - 28);
+    cycles.push({
       cycleLength: 28,
       periodLength: 6,
-      startDate: "2023-06-30",
-    },
-  ] satisfies Cycle[]);
+      startDate: date.toString(),
+    });
```

Comment:
You remove but forgot to return `satisfies` here

Reply:
In my situation it isn`t needed. I removed this.